src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2698,13 +2698,17 @@
         "kind": {
           "description": "A string that indicates where the suppression is persisted.",
           "enum": [
-            "suppressedInSource",
-            "suppressedExternally",
-            "underReview",
-            "suppressionRejected"
+            "inSource",
+            "external"
           ]
         },
 
+        "state": {
+          "description": "A string that indicates the state of the suppression. Well-known values are `accepted`, `underReview` and `rejected`.",
```

Comment:
>description [](start = 11, length = 11)

shouldn't this be an enum? not an open ended string?

Reply:
the language in spec issue text seems to indicate an open ended string, but i could be wrong. I will check with Larry!

---
In reply to: [277074605](https://github.com/Microsoft/sarif-sdk/pull/1442#discussion_r277074605) [](ancestors = 277074605)

Reply:
This should be enum!

---
In reply to: [277089440](https://github.com/Microsoft/sarif-sdk/pull/1442#discussion_r277089440) [](ancestors = 277089440,277074605)